graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/internal/InputArgumentTest.kt

```diff
@@ -1395,6 +1400,93 @@ internal class InputArgumentTest {
         verify { applicationContextMock.getBeansWithAnnotation(DgsComponent::class.java) }
     }
 
+    @Test
+    fun `Kotlin Optional enum @InputArgument`() {
+        val schema = """
+            type Query {
+                hello(type:GreetingType): String
+            }
+            
+            enum GreetingType {          
+                FRIENDLY
+                POLITE
+            }
+        """.trimIndent()
+
+        val fetcher = object : Any() {
+            @DgsData(parentType = "Query", field = "hello")
+            fun someFetcher(@InputArgument type: Optional<KGreetingType>): String {
+                val typeValue: Any = type.get()
+                if (typeValue !is KGreetingType) {
```

Comment:
Why not just use a fluent assertJ check? 
```
assertThat(type).isNotEmpty.get().isInstanceOf(KGreetingType::class.java)
```

Will result in
```
Expecting:
  <"B">
to be an instance of:
  <com.netflix.graphql.dgs.internal.kotlin.test.KGreetingType >
but was instance of:
  <java.lang.String>
```

Reply:
yep, good idea. Updated.